Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine3.12
+FROM python:3.9-alpine
```

Comment:
any reason why not to use alpine 3.12 ?

Reply:
Because I don't want to update the version number every other month. `3.9-alpine` will just use the latest stable with Python 3.9.

Reply:
Ah sorry I've read that line as `alpine:3.7` changed to `alpine:3.9`..

It might be worth to switch to a retagged python image from our quay.io account, but not blocking here.